node/src/api/server.rs

```diff
@@ -20,6 +21,12 @@ where F: FnOnce(&mut web::ServiceConfig) + Send + Clone + 'static {
         config.actix.addr().to_socket_addrs()?.next().unwrap()
     );
 
+    let mut consensus_processor = ConsensusProcessor::new(config.clone());
+    let (consensus_sender, consensus_receiver) = mpsc::channel::<()>();
+    actix_rt::spawn(async move {
```

Comment:
Leaving note for the future
```suggestion
    // TODO: spawn consensus processors in separate Runtime
    actix_rt::spawn(async move {
```

Reply:
Added it above